pkg/utils/util.go

```diff
@@ -338,3 +339,13 @@ func ApiextensionsJsonToKyvernoConditions(original apiextensions.JSON) (interfac
 	}
 	return nil, fmt.Errorf("error occurred while parsing %s: %+v", path, err)
 }
+
+func OverrideRuntimeErrorHandler() {
+	if len(runtime.ErrorHandlers) > 0 {
+		runtime.ErrorHandlers[0] = func(err error) {}
+	} else {
+		runtime.ErrorHandlers = []func(err error){
+			func(err error) {},
+		}
+	}
```

Comment:
This will override all error handlers....are we sure we want to do that? Or do we want to keep the backoff handler?

Is this used anywhere else?

Also, instead of a empty error handler, I suggest logging the error message at a higher level.

Reply:
I haven't found anywhere in code where we use runtime ErrorHandlers.
Correct, instead of  empty error handler, we can log error higher level.

Reply:
done